### DISCUSSION TITLE
能在发布版本时注明一下支持的最小最大zotero版本吗

### DISCUSSION THREAD

**Discussion by wwdok** [windingwind/zotero-better-notes#746] [Q&A]
感谢作者开源的一系列zotero插件，我在安装插件过程中，遇到了报错：“无法安装插件“%S”。它可能无法与该版本的 Zotero 兼容”，我的zotero版本是6.0.27，这是从官网下载默认的版本，后面发现我得安装比较早的插件版本，但太早的版本我又怕缺少一些功能或有bug，所以我想安装支持zetero 6.0.27的最新的better-notes版本，但我又不知道对应哪个版本的better-notes

**Comment by windingwind** [comment]
见readme#install。不会也没必要每次都标注
